UPGRADE.md

```diff
@@ -1,5 +1,15 @@
 # Upgrade
 
+## dev-master
+
+### Filter
+
+Update the schema `app/console doctrine:schema:update --force` and run following SQL-Statement:
+ 
+```sql
+UPDATE re_conditions SET value = CONCAT('"', value, '"') WHERE value NOT LIKE '"%"'
```

Comment:
i think we should not introduce new dependencies in master to make the upgrade easier /cc @chirimoya @danrot what did yo think?